application.php

```diff
@@ -17,6 +17,7 @@
         isset($_SERVER['jira_build_message']) ? $_SERVER['jira_build_message'] : null,
         isset($_SERVER['jira_url']) ? $_SERVER['jira_url'] : null,
         isset($_SERVER['jira_project']) ? $_SERVER['jira_project'] : null,
+        isset($_SERVER['jira_issue_key']) ? $_SERVER['jira_issue_key'] : null,
```

Comment:
Let's go for this and I will make a PR move the parameter as an option instead. I'll do this before pushing the the bitrise steplib